kbase-extension/static/kbase/js/common/cellComponents/fieldTableCellWidget.js

```diff
@@ -74,17 +78,14 @@ define([
                         id: ids.inputControl,
                         dataElement: 'input-control',
                     }),
-                    div(
-                        {
-                            class: `${messageBaseClass} hidden`,
-                            dataElement: 'message-panel',
-                        },
-                        [
-                            div({
-                                class: `${messageBaseClass}__body`,
-                            }),
-                        ]
-                    ),
+                    div({
+                        class: `${messageBaseClass} hidden`,
+                        dataElement: 'message-panel',
+                    }),
+                    div({
```

Comment:
A message panel and duplicate value panel should both be allowed at the same time. This seemed easier than mucking about with adding and removing from the DOM and making sure the order was always right.